lib/hooks/queries/useTransactionHistory.ts

```diff
@@ -68,7 +70,7 @@ export const useTransactionHistory = (address: string) => {
       if (isIndexedSdk(sdk) && address) {
         const { historicalAssets } = await sdk.indexer.client.request<{
           historicalAssets: {
-            assetId: string;
+            assetId: string | AssetId;
```

Comment:
Are you sure this can be both types?

Reply:
I was testing something out and forgot to remove this. I think it can stay as `string`